File: NashvillePublicArtCapstone/NashvillePublicArtCapstone/Controllers/ProfileController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace NashvillePublicArtCapstone.Controllers
{
    public class ProfileController : Controller
    {
        public ActionResult Profile()
        {
            ViewBag.Title = "Profile Page";

            return View();
        }
    }
}

File: NashvillePublicArtCapstone/NashvillePublicArtCapstone/app/ProfileController.js
(function () {
    angular.module("NashvillePublicArtCapstone")
        .controller("ProfileController", ProfileController);

    ProfileController.$inject = ["$scope"];

    function ProfileController($scope) {
        $scope.testUserList = "Testing user's list page";
        console.log($scope);
    }
})();